core/domain/fs_domain_test.py

```diff
@@ -105,16 +107,108 @@ def test_independence_of_file_systems(self):
             fs2.get('abc.png')
 
 
-class DiskBackedFileSystemTests(test_utils.GenericTestBase):
+class DiskBackedFileSystemUnitTests(test_utils.GenericTestBase):
     """Tests for the disk-backed file system."""
 
+    def setUp(self):
+        super(DiskBackedFileSystemUnitTests, self).setUp()
+        self.user_id = 'abc@example.com'
```

Comment:
This looks like an email. Did you mean self.user_email?

Reply:
It was self.user_id in the tests firstly written in this file, so I just kept that.

Also, I think id can refer to mail as well. Let me know your view, I'll follow that.

Reply:
Hmmm, but in other test files, the id refers to the database id, while email refers to the email ID. Can we also edit other occurrences in this file?

Reply:
Done.